docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx

```diff
@@ -15,7 +15,7 @@ The table below gives a quick overview of the benefits of Teleport Enterprise.
 | Teleport Enterprise Feature | Description |
 | - | - |
 | [Single Sign-On (SSO)](#sso) | Allows Teleport to integrate with existing enterprise identity systems. Examples include Active Directory, GitHub, Google Apps and numerous identity middleware solutions like Auth0, Okta, and so on. Teleport supports SAML and OAuth/OpenID Connect protocols to interact with them. |
-| [Access Requests](../../access-controls/access-requests.mdx) | User interface for teams to create and review requests to access infrastructure with escalated privileges. |
+| [Access Requests](../../access-controls/access-requests.mdx) | equest elevated access to roles or specific resources. |
```

Comment:
Just noticed that there's a spelling issue here @stevenGravy

Reply:
Thanks, copy and paste error, corrected, thanks!